### PR TITLE
Support for "enrichment" and wildcard "by" options.

### PR DESCRIPTION
**Pull request by armiol**
Summary of changes:

* Illustrate `enrichment = "TargetEnrichment"` and `(by) = "*.event_id"` options in a core-java test.
* Depend on Spine protobuf-plugin of `0.6.6-SNAPSHOT` version.

NOTE: `core-java` version is NOT increased, so that this PR could be merged to whatever codebase version in `master`.

### CODE REVIEW

**Comment by armiol** [comment]
@alexander-yevsyukov PTAL.

File: client/src/main/proto/spine/event_annotations.proto [hunk 1]
```diff
@@ -85,7 +85,17 @@ extend google.protobuf.MessageOptions {
...
     // in the enrichment process must have the same names.
     //
-    string enrichment = 57126;
+    // In the enrichment message a wildcard `by` option syntax may be used to reference more than a single
+    // target events to enrich.
+    //
+    // For example:
+    //
+    // message EnrichmentForSeveralEvents {
+    //     string str = 1 [(by) = "*.user_id"];
```
Comment (alexander-yevsyukov) [inline client/src/main/proto/spine/event_annotations.proto]: Can we have `user_name` instead of `str`? Here and other such places, please.

Comment (armiol) [inline client/src/main/proto/spine/event_annotations.proto]: Done.

**Comment by armiol** [comment]
@alexander-yevsyukov PTAL.

**Pull request merged**